### PR TITLE
Fix property names to be related to response from riot valorant api

### PR DESCRIPTION
**Pull request by mohasaid**
Fix issue related to #32

### CODE REVIEW

File: src/main/java/no/stelar7/api/r4j/pojo/val/matchlist/MatchReference.java [hunk 1]
```diff
@@ -6,26 +6,26 @@
...
 {
     private static final long serialVersionUID = -5301457261872587385L;
-    
+
     private String matchId;
-    private Long   gameStartTime;
-    private String teamId;
-    
+    private Long   gameStartTimeMillis;
+    private String queueId;
```
Comment (stelar7) [inline src/main/java/no/stelar7/api/r4j/pojo/val/matchlist/MatchReference.java:12]: Type this as `GameQueueType`

File: src/main/java/no/stelar7/api/r4j/pojo/val/matchlist/MatchReference.java [hunk 2]
```diff
@@ -6,26 +6,26 @@
 public class MatchReference implements Serializable
 {
     private static final long serialVersionUID = -5301457261872587385L;
```
Comment (stelar7) [inline src/main/java/no/stelar7/api/r4j/pojo/val/matchlist/MatchReference.java:8]: Generate a new `serialVersionUID`

**Review by stelar7 — CHANGES_REQUESTED** [review]

File: src/main/java/no/stelar7/api/r4j/pojo/val/matchlist/MatchReference.java (hunk 1 above)
Comment (mohasaid) [reply src/main/java/no/stelar7/api/r4j/pojo/val/matchlist/MatchReference.java]: done!

File: src/main/java/no/stelar7/api/r4j/pojo/val/matchlist/MatchReference.java [hunk 3]
```diff
@@ -1,31 +1,33 @@
...
+
+    public Long getGameStartTimeMillis()
     {
-        return gameStartTime;
+        return gameStartTimeMillis;
     }
-    
-    public String getTeamId()
+
+    public GameQueueType getQueueId()
```
Comment (stelar7) [inline src/main/java/no/stelar7/api/r4j/pojo/val/matchlist/MatchReference.java:26]:
```suggestion
    public GameQueueType getQueue()
```

**Pull request merged**